RELEASE.md

```diff
@@ -0,0 +1,18 @@
+Release type: minor
+
+Support for schema-extensions in subscriptions.
```

Comment:
can we add an example of how this could be used? so people know what kind of functionality this enables?